scripts/integration-tests/utils/set-babel-resolutions.js

```diff
@@ -0,0 +1,17 @@
+import fs from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const packageJSONPath = path.resolve(process.cwd(), "./package.json");
+const content = JSON.parse(fs.readFileSync(packageJSONPath));
+
+const resolutions = content.resolutions || {};
+for (const name of fs.readdirSync(path.join(__dirname, "../../../packages"))) {
+  if (!name.startsWith("babel-")) continue;
+  resolutions[name.replace("babel-", "@babel/")] = "*";
+}
+content.resolutions = resolutions;
+
+fs.writeFileSync(packageJSONPath, JSON.stringify(content, undefined, 2));
```

Comment:
Can we merge this script with `scripts/integration-tests/utils/bump-babel-dependencies.js`? I think they serve the same purpose: They both update the `@babel/*` deps so that we can test against the main branch.